otel-integration/k8s-helm/README.md

```diff
@@ -1778,6 +1778,16 @@ presets:
     enabled: true
 ```
 
+#### Known errors
+
+When running on Windows, you might see the "failed getting host info" error. This is expected behavior because the collector attempts to retrieve Windows metadata from the Windows Registry, which is only possible when running from HostProcess Windows containers.
```

Comment:
I would add to this that it has no negative impact in functionality of the Collector or OpAMP in any way.